src/WidenImage.css

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
I made an attempt to do so after our last chat at the onsite/offsite but ran into a few complications. Then I did an audit on the iOS transform less files and didn't see any actual less syntax, so I was thinking we could go with plain css for now and re-evaluate later as needed if that sounded ok?